mslib/msui/wms_control.py

```diff
@@ -1547,7 +1547,10 @@ def display_retrieved_image(self, imgs, legend_imgs, layer, style, init_time, va
         self.view.draw_image(self.squash_multiple_images(imgs))
         self.view.draw_legend(self.append_multiple_images(legend_imgs))
         style_title = self.multilayers.get_current_layer().get_style()
-        self.view.draw_metadata(title=self.multilayers.get_current_layer().layerobj.title,
+        title = self.multilayers.get_current_layer().layerobj.title
```

Comment:
with this complexity and repition, I argue that this should be refactored to a get_title() method of self.multilayers